src/parser/Statement.ts

```diff
@@ -3415,3 +3415,68 @@ export class ContinueStatement extends Statement {
         return this.tokens.continue?.leadingTrivia ?? [];
     }
 }
+
+
+export class TypeCastMStatement extends Statement {
```

Comment:
Let's call this `TypecastStatement`. While the current implementation is restricted to `m`, we might expand this in the future (casting global functions, consts, etc...), so I want to leave the door open to support those without breaking AST changes.

Reply:
done